.jenkins/pipelines/p1-tests.Jenkinsfile

```diff
@@ -33,7 +32,6 @@ pipeline {
                     // This is used to test on specific VM generations
                     IGNORE_VM_GENERATIONS = []
```

Comment:
IGNORE_VM_GENERATIONS and the check for ICELAKE_VM can be removed too as it has no functionality.

Reply:
IGNORE_VM_GENERATIONS is also on line 97